questions.js

```diff
@@ -286,6 +286,7 @@ class Question {
             this.updateAnswer();
             config.questionAnswers[this.id] = this.getAnswerInput();
             saveConfig();
+            if(e.key=="Enter") document.getElementById("forward").click()
```

Comment:
would be better if it didn't rely on simulating clicking the button and directly went to the next page using the existing code